ui/app/AppLayouts/Profile/controls/AccountDelegate.qml

```diff
@@ -0,0 +1,22 @@
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core 0.1
+
+
+StatusListItem {
+    property var account
+    title: account.name
+    subTitle: account.address
+    image.source: "..."
```

Comment:
why this? just dont assign it as it is an identicon

Reply:
If I remove this, the identicon is gone

Reply:
we need to use it as icon.isLetterIdenticon: true I think then we dont need to do this

Reply:
Thanks it works. Updated